sound/soc/sof/intel/hda-loader.c

```diff
@@ -90,7 +90,8 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, const void *fwdata,
 	/* step 1: power up corex */
 	ret = hda_dsp_core_power_up(sdev, chip->cores_mask);
 	if (ret < 0) {
-		dev_err(sdev->dev, "error: dsp core 0/1 power up failed\n");
+		if (iteration == HDA_FW_BOOT_ATTEMPTS)
+			dev_err(sdev->dev, "error: dsp core 0/1 power up failed\n");
```

Comment:
@ranj063  Hmm, how about making these all dev_warn() (without "error" in message) in cl_dsp_init. It's still suspicious that a boot attempt fails, so I think a warning is appropriate?